core/src/commonTest/kotlin/com/sourcepoint/mobile_core/CoordinatorTest.kt

```diff
@@ -87,9 +87,9 @@ class CoordinatorTest {
     @Test
     fun noMessagesShouldAppearAfterAcceptingAll() = runTest {
         assertEquals(3, coordinator.loadMessages(authId = null, pubData = null).size)
-        coordinator.reportAction(SPAction(SPActionType.AcceptAll, SPCampaignType.Gdpr))
-        coordinator.reportAction(SPAction(SPActionType.AcceptAll, SPCampaignType.Ccpa))
-        coordinator.reportAction(SPAction(SPActionType.AcceptAll, SPCampaignType.UsNat))
+        coordinator.reportAction(SPAction(SPActionType.AcceptAll, SPCampaignType.Gdpr, pmPayload = "{}"))
+        coordinator.reportAction(SPAction(SPActionType.AcceptAll, SPCampaignType.Ccpa, pmPayload = "{}"))
+        coordinator.reportAction(SPAction(SPActionType.AcceptAll, SPCampaignType.UsNat, pmPayload = "{}"))
```

Comment:
@Nevazhnovu does passing `""` to `pmPayload` work, instead of `"{}"`?

Reply:
This one was discussed in Slack